### PR TITLE
IN-1864 views.view.member_by_office.yml

### PR DESCRIPTION
**Pull request by jdearie**
Create view that pulls email addresses of group members. Include a filter that pulls all email addresses from all groups owned by that office.

### CODE REVIEW

File: config/sync/views.view.member_by_office.yml
Comment (jmarruff) [inline config/sync/views.view.member_by_office.yml:1]: I was messing with this yesterday, but for some reason the view didn't allow for seeing the email. I noticed that you are using role/authenticated - did that actually work? I created a view for Christina, but allowing authenticated didn't seem work for me so just wondering if that worked on you local.

Comment (jmarruff) [reply config/sync/views.view.member_by_office.yml:1]: I should add that I could see them as an admin.

Comment (jdearie) [reply config/sync/views.view.member_by_office.yml:1]: I discovered a very specific permission that needed to be granted to anonymous and authenticated users:
_View user email addresses_

Comment (jmarruff) [reply config/sync/views.view.member_by_office.yml:1]: I figured it may not work without some permissions adjustments.

**Pull request merged**